templates/components/kanban/item_panels/default_panel.html.twig

```diff
@@ -0,0 +1,106 @@
+{#
+ # ---------------------------------------------------------------------
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ # Copyright (C) 2015-2021 Teclib' and contributors.
+ #
+ # http://glpi-project.org
+ #
+ # based on GLPI - Gestionnaire Libre de Parc Informatique
+ # Copyright (C) 2003-2014 by the INDEPNET Development Team.
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # GLPI is free software; you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation; either version 2 of the License, or
+ # (at your option) any later version.
+ #
+ # GLPI is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with GLPI. If not, see <http://www.gnu.org/licenses/>.
+ # ---------------------------------------------------------------------
+ #}
+
+{% import 'components/form/fields_macros.html.twig' as fields %}
+
+<div class="item-details-panel" data-itemtype="{{ itemtype }}" data-items_id="{{ item_fields.id }}">
+   <div class="card d-flex flex-column h-100">
+      <div class="card-header d-block">
+         <h5 class="card-title">
+            <a href="{{ itemtype|itemtype_form_path(item_fields.id) }}">
+               <i class="{{ itemtype|itemtype_icon }}" title="{{ itemtype|itemtype_name }}"></i>
+               {{ item_fields.name }}
+            </a>
+            <button type="button" class="btn-link" onclick="$(this).closest('.item-details-panel').remove()"><i class="fas fa-times"></i></button>
+         </h5>
+         <h6 class="card-subtitle">
+            {% if item_fields.is_milestone %}
+                  <i class="fas fa-map-signs me-2"></i>{{ __('Milestone') }}
+            {% endif %}
+         </h6>
+      </div>
+      <div class="card-body overflow-auto">
+         {% macro print_hook_section(name, itemtype, items_id) %}
+            {% set content = call_plugin_hook_func(name, {
+               "itemtype": itemtype,
+               "items_id": items_id
+            }, true) %}
+            {% if content is not empty and content['content']|trim is not empty %}
+               {{ content['content']|raw }}
+               <hr>
+            {% endif %}
+         {% endmacro %}
+         {{ _self.print_hook_section('pre_kanban_panel_content', itemtype, item_fields.id) }}
+         {{ _self.print_hook_section('pre_kanban_panel_main_content', itemtype, item_fields.id) }}
+
+         {% set preview = item_fields.content|length > 1000 ? item_fields.content|slice(0, 1000) ~ ' (...)' : item_fields.content %}
+         {{ fields.textareaField('content', preview, '', {
```

Comment:
Why displaying it in a disabled textarea ?

Reply:
It wasn't planned to allow editing the name or content from the panel itself. It is only to view basic details and manage team members.